tests/protocols/unlock_test.py

```diff
@@ -205,23 +203,27 @@ def testMultipleChoice_multipleFailsBeforeSuccess(self):
             else:
                 self.validateRecord(attempt, answer=self.SHORT_ANSWER, correct=True)
 
+    EVAL_ANSWER = ['[1, 2, 3, 4]']
+    CORRECT_EVAL = ['[1,2,3,2+2]']
+    INCORRECT_EVALS = ['[4,3,2,1]', '{1,2,3,4}', '[1,2,3,8/2]']
```

Comment:
For future reference: `ast.literal_eval` supports addition/subtraction of integers (but not other operations/other types)

I don't think this is worth a comment in the test.

Also might be worth a shot to give it something it just fail on - like `raise Exception("test")` (which throws a SyntaxError)